ts/src/binance.ts

```diff
@@ -10082,17 +10137,24 @@ export default class binance extends Exchange {
          * @param {string} [method] method name to call, "positionRisk", "account" or "option", default is "positionRisk"
          * @returns {object[]} a list of [position structure]{@link https://docs.ccxt.com/#/?id=position-structure}
          */
-        const defaultValue = this.safeString (this.options, 'fetchPositions', 'positionRisk');
         let defaultMethod = undefined;
-        [ defaultMethod, params ] = this.handleOptionAndParams (params, 'fetchPositions', 'method', defaultValue);
+        [ defaultMethod, params ] = this.handleOptionAndParams (params, 'fetchPositions', 'method');
+        if (defaultMethod === undefined) {
+            const option = this.safeValue (this.options, 'fetchPositions');
+            if (typeof option === 'string') {
+                defaultMethod = option;
+            } else {
+                defaultMethod = 'positionRisk';
```

Comment:
@ttodua why this weird handling ? we were already using `handleOptionAndParams` so why add this extra check here?

Reply:
because in these lines there are proper support of two different type of options